### PR TITLE
[9.0] extract analytic base department from OCA/department

### PR DESCRIPTION
**Pull request by yvaucher**
Moving OCA/department#56 here

### CODE REVIEW

File: analytic_base_department/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2011 Joël Grand-Guillaume (Camptocamp)
+# © 2013 Daniel Reis (Camptocamp)
```
Comment (dreispt) [inline analytic_base_department/__init__.py]: Not sure how this got here, but I'm not from Camptocamp; please remove

Comment (yvaucher) [inline analytic_base_department/__init__.py]: Damn I fixed it in the other PR but forgot it here :-/

**Comment by yvaucher** [comment]
@dreispt fixed

File: analytic_base_department/README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project/issues>`_. In case of trouble, please
```
Comment (yvaucher) [inline analytic_base_department/README.rst]: s/project/account-analytic

File: analytic_base_department/README.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Project Department Categorization (`project_departement`)
+- Project Task specific Department (`project_task_departement`)
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
```
Comment (yvaucher) [inline analytic_base_department/README.rst]: replace 140 by the right repository id

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by guewen** [comment]
:+1:

**Comment by moylop260** [comment]
👍

**Comment by mariano-aferrari** [comment]
@copilot please review

**Pull request merged**